### PR TITLE
updata ds docs

### PR DESCRIPTION
**Pull request by mirii1994**

### CODE REVIEW

File: docs/data-sources/archive_logs.md [hunk 1]
```diff
@@ -9,24 +9,16 @@ Provides a Logz.io archive logs resource. This can be used to create and manage
...
-* `azure_blob_storage_settings.client_id` - (String) Azure application (client) ID. The Client ID of the AD app.
-* `azure_blob_storage_settings.client_secret` - (String) Azure client secret.
-* `azure_blob_storage_settings.account_name` - (String) Azure Storage account name.
-* `azure_blob_storage_settings.container_name` - (String) Name of the container in the Storage account.
-* `azure_blob_storage_settingspath` - (String) Optional virtual sub-folder specifiying a path within the container.
+* `credentials_type` - (String) Applicable when `storage_type` is `S3`. Specifies which credentials will be used for authentication.
+* `s3_path` - (String) Applicable when `storage_type` is `S3`. Specify a path to the **root** of the S3 bucket.
+* `s3_iam_credentials_arn` - (String) Applicable when `storage_type` is `S3`. Amazon Resource Name (ARN) to uniquely identify the S3 bucket.
+* `aws_access_key` - (String) Applicable when `storage_type` is `S3`. AWS access key.
+* `tenant_id` - (String) Applicable when `storage_type` is `BLOB`. Azure Directory (tenant) ID. The Tenant ID of the AD app.
```
Comment (Doron-Bargo) [inline docs/data-sources/archive_logs.md:18]: Why we removed the prefix? i think we need to keep the prefix for the relevant cloud vendor

**Review by Doron-Bargo — CHANGES_REQUESTED** [review]
Please check my comments

File: docs/data-sources/archive_logs.md (hunk 1 above)
Comment (mirii1994) [reply docs/data-sources/archive_logs.md:18]: The API changes made it impossible to keep in that structure. I had to make the change.

**Review by nico-shishkin — APPROVED** [review]

**Review by Doron-Bargo — APPROVED** [review]
Great

**Pull request merged**